grails-app/services/org/pih/warehouse/fulfillment/FulfillmentService.groovy

```diff
@@ -70,4 +101,252 @@ class FulfillmentService {
 
         return command
     }
+
+    StockMovement createOutbound(ImportPackingListCommand command) {
+        // First create a requisition
```

Comment:
IMO you probably don't need a lot of these comments since they're just reiterating what the method names are (which means you have good method names). My general approach is to have self-documenting code (ie writing code in a way that makes the functionality obvious by just reading the code) and then add comments if it's still unclear.

Not a big deal though and better to be overly verbose than have very unclear code.

Reply:
@EWaterman I also had a doubt whether to put those comments or not, but thought that maybe it's better to explain the iteration explicitly.